<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BabelClass</class>
 <widget class="QMainWindow" name="BabelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Babel</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackWindows">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="ServerConnect">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QGroupBox" name="babelPic">
             <property name="minimumSize">
              <size>
               <width>271</width>
               <height>241</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>271</width>
               <height>241</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="title">
              <string>Babel</string>
             </property>
             <widget class="QLabel" name="babelDisplay">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>24</y>
                <width>251</width>
                <height>207</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>251</width>
                <height>207</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>qt-logo.png</pixmap>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QWidget" name="widget_2" native="true">
             <property name="minimumSize">
              <size>
               <width>271</width>
               <height>171</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>271</width>
               <height>171</height>
              </size>
             </property>
             <widget class="QLineEdit" name="serverField">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>30</y>
                <width>221</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>192.168.0.1</string>
              </property>
             </widget>
             <widget class="QLabel" name="serverLabel">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>10</y>
                <width>221</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Server address :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="portField">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>80</y>
                <width>221</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>20701</string>
              </property>
             </widget>
             <widget class="QLabel" name="portLabel">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>60</y>
                <width>221</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Port :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QPushButton" name="connectButton">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>120</y>
                <width>201</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Connection</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ServerLogin">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_3" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QGroupBox" name="LoginBox">
             <property name="minimumSize">
              <size>
               <width>251</width>
               <height>161</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>251</width>
               <height>161</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="title">
              <string>Login</string>
             </property>
             <widget class="QLabel" name="LoginLabel">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>20</y>
                <width>91</width>
                <height>20</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="loginField">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>40</y>
                <width>133</width>
                <height>20</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
             <widget class="QPushButton" name="loginButton">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>130</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="passwordField">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>100</y>
                <width>133</width>
                <height>20</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="PasswordLabel">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>80</y>
                <width>67</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="RegisterBox">
             <property name="minimumSize">
              <size>
               <width>251</width>
               <height>181</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>251</width>
               <height>181</height>
              </size>
             </property>
             <property name="title">
              <string>Register</string>
             </property>
             <widget class="QLabel" name="RLoginLabel">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>20</y>
                <width>81</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QLabel" name="RPwdLabel">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>50</y>
                <width>121</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="RLoginField">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>40</y>
                <width>133</width>
                <height>20</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
             <widget class="QLineEdit" name="RPwdField">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>80</y>
                <width>133</width>
                <height>20</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
             <widget class="QLineEdit" name="RCPwdField">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>120</y>
                <width>133</width>
                <height>20</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
             <widget class="QLabel" name="RCPwdLabel">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>100</y>
                <width>121</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Confirm password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QPushButton" name="registerButton">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>150</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Register</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Identified">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Contact List&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListView" name="listView">
          <property name="minimumSize">
           <size>
            <width>311</width>
            <height>371</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="DirectCall">
          <property name="minimumSize">
           <size>
            <width>291</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>291</width>
            <height>51</height>
           </size>
          </property>
          <property name="title">
           <string>Direct Call</string>
          </property>
          <widget class="QPushButton" name="callButton">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>20</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Call</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="PortClientLine">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>20</y>
             <width>51</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>20705</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="IpClientLine">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>133</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Calling">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="inProgress">
          <property name="text">
           <string>Call in Progress</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="endCallButton">
          <property name="text">
           <string>Raccrocher</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>347</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="babel.qrc"/>
 </resources>
 <connections/>
</ui>
